### PR TITLE
Added uGAS-0921 to the assets.json

### PR DESCRIPTION
**Pull request by snake-poison**
### Description of changes proposed in this pull request:

- Add uGas-0921 to the assets.json

### Checks:

- [X] Have you followed the guidelines in our Contributing document?
- [X] Have you checked to ensure there aren't other open [Pull Requests](../../pulls) for the same update/change?

### CODE REVIEW

**Comment by snake-poison** [comment]
Might want to wait for some liquidity before accepting this PR and maybe we want to modify the "coming soon" graphics and text. Just wanted to get the ball rolling.

**Comment by ethedev** [comment]
@ElmoArmy @sali0 either lowercase all these addresses, either @sali0 add handling for [this](https://github.com/yam-finance/degenerative-v2/issues/196), and once any is good to go we only can merge once the website is ready with:
- the new twap endpoint
- the "new" tag on the new ugas synth

File: src/assets/assets.json [hunk 1]
```diff
@@ -140,6 +140,23 @@
...
+          "address": "0xfc10b3A8011B00489705EF1Fc00D0e501106cB1D",
+          "decimals": 18
+        },
+        "emp": {
+          "address": "0xcA2531b9CD04daf0c9114D853e7A83D8528f20bD"
+        },
+        "pool": {
+          "address": "0x5CCD155ad26B74913ed6266A516A085A2343D426",
+          "location": "sushi"
+        }
```
Comment (ethedev) [inline src/assets/assets.json]:
```suggestion
        "token": {
          "address": "0xfc10b3a8011b00489705ef1fc00d0e501106cb1d",
          "decimals": 18
        },
        "emp": {
          "address": "0xca2531b9cd04daf0c9114d853e7a83d8528f20bd"
        },
        "pool": {
          "address": "0x5ccd155ad26b74913ed6266a516a085a2343d426",
          "location": "sushi"
        }
```

**Comment by ethedev** [comment]
@sali0 i updated it here, just fix handling it on this #196

**Pull request merged**